rqt_reconfigure/src/rqt_reconfigure/paramedit_widget.py

```diff
@@ -109,15 +109,9 @@ def show_reconf(self, dynreconf_widget):
             #    self.vlayout.addWidget(v)
 
         # Add color to alternate the rim of the widget.
-        try:
-            LayoutUtil.alternate_color(self._dynreconf_clients.itervalues(),
-                [self.palette().window().color().lighter(125),
-                self.palette().window().color().darker(125)]) # Qt5
-        except AttributeError:
-            LayoutUtil.alternate_color(self._dynreconf_clients.itervalues(),
-                [self.palette().background().color().lighter(125),
-                self.palette().background().color().darker(125)]) # Qt4
-
+        LayoutUtil.alternate_color(self._dynreconf_clients.itervalues(),
+            [self.palette().window().color().lighter(125),
+            self.palette().window().color().darker(125)]) # Qt5
```

Comment:
I think we should remove the reference to Qt5 - we've established that `window()` is used in Qt4 as well.